### PR TITLE
refactor: move Dgraph schema generation to services package

### PR DESCRIPTION
**Pull request by rhyslbw**
# Context
Generating Dgraph schema is better suited to be sourced in `cardano-graphql-services`, since this is where the projection code is.

# Proposed Solution
Move the code, scripts, and documentation

### CODE REVIEW

**Review by mkazlauskas — CHANGES_REQUESTED** [review]
I'm not sure about moving `yarn generate`. It generates `sdk.ts` which is part of client-side scope that belongs in `cardano-graphql`. Also, one of the inputs to codegen is `operations/` graphql files for queries, which definitely belong in `cardano-graphql`.

At the same time, it would be cleaner if `cardano-graphql` didn't need to have a docker-compose.yml.

Perhaps we should both:
- Keep codegen in `cardano-graphql`
- Keep docker-compose under `cardano-graphql-services`, documenting in `cardano-graphql` that you need to have it running for codegen

File: packages/cardano-graphql/codegen.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-schema: ./dist/dgraph.graphql
+schema: http://localhost:8080/
```
Comment (mkazlauskas) [inline packages/cardano-graphql/codegen.yml:1]: Nice, have you tested this?

**Review by mkazlauskas — APPROVED** [review]
Please confirm that this is the new workflow after updating schema (might be worth documenting too?):
- `cd packages/cardano-graphql-services && yarn build && yarn build:schema:dgraph`
- `cd packages/cardano-graphql` && yarn build && yarn generate`

**Comment by rhyslbw** [comment]
> Please confirm that this is the new workflow after updating schema (might be worth documenting too?):

eb5aea45ed6556a115b057a4b9e2972470472253 includes those docs, plus starting and stopping the Docker Compose stack.

**Review by jvieiro — APPROVED** [review]

**Pull request merged**